os/board/rtl8720e/src/component/wifi/api/wifi_conf_inter.c

```diff
@@ -0,0 +1,63 @@
+//----------------------------------------------------------------------------//
```

Comment:
We'll need an open source license here please.

Reply:
Dear @amogh-samsung,
I checked the other files in this layer, seems all don't provide license. These are the API files from our sdk, so maybe don't need license here. Is that okay for customer side?
Thank You.